fourkitchens/suhumsci
# Shs 4932 pager headings

## Summary
Alters the standard pager `h4` to `h2`

## Steps to Test
1. Go to a page with a pager such as `default-views/news`
2. Inspect the pager and see that there is a visually hidden heading that says 'pagination'. It is supposed to be an `h2`

### Notes
- I chose the base branch of `develop` because it looks like we need the latest release. 
- I tested the page before any work was done with an accessibility tool. This doesn't 'increase' or add to the accessibility as the old `h4` was valid. However, I decided to go ahead and change it to an `h2` so that it's consistent with all the other supplementary `nav` elements (they all use `h2`).
